config/src/config/execution_config.rs

```diff
@@ -47,7 +47,7 @@ impl Default for ExecutionConfig {
             // Default value of 30 seconds for the network timeout.
             network_timeout_ms: 30_000,
             // Sequential execution by default.
-            concurrency_level: 1,
+            concurrency_level: 8,
```

Comment:
update the comments?

Reply:
Hmm, will it cause unit tests to start using too much of resources on CI destabilizing them?

Reply:
overall CPU utilization should be the same (even on the 1s intervals)? it just might be more spiky - larger but shorter?

I've updated smoke tests to concurrency_level=1 just in case, but for the rest I think should be fine. let's see what CI says on this PR